import Exception from "../core/Exception";
export default class TestThrowException extends Exception {
    getName(): string;
    getMessage(): string;
}
